shardingsphere-scenario/config/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '2.1'
+services:
+  mysql-server:
```

Comment:
I suggest that you should remove `mysql-server` configure item if you don't use mysql